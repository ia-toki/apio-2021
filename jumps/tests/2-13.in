209636825ecd2c38299498cd26890ef71dd128c2
200 200
29 2 3 4 5 122 7 8 48 10 11 141 105 14 15 16 35 139 19 20 21 197 23 24 25 26 76 28 1 30 91 154 33 34 114 156 37 38 39 52 103 42 43 44 118 46 64 56 112 50 51 40 53 54 55 9 57 58 59 60 61 62 198 47 12 130 67 68 181 70 71 72 73 170 45 49 77 78 79 97 180 82 193 84 85 86 87 131 89 90 27 92 93 138 95 173 69 98 99 100 101 102 41 157 13 106 135 108 137 110 111 151 83 65 115 196 160 140 187 120 121 6 123 36 125 126 194 164 129 166 147 116 17 134 168 159 109 133 18 75 188 142 22 132 153 163 127 199 149 117 190 152 185 96 155 128 104 158 172 136 113 162 94 150 165 148 200 107 169 161 186 124 195 174 175 176 177 178 192 146 80 167 182 74 143 66 32 81 189 144 145 179 119 88 31 183 171 63 191 184
0 0 199 199
149 189 197 197
50 127 198 198
74 82 182 198
74 75 148 183
10 176 192 199
123 163 168 181
24 26 87 187
10 40 165 176
110 198 199 199
4 8 60 126
19 182 195 197
27 30 106 183
3 125 189 198
48 93 188 188
190 196 198 198
107 140 153 167
3 5 125 136
9 69 163 168
3 32 34 137
10 54 65 106
142 180 189 191
2 87 118 138
44 116 146 185
20 115 143 180
2 4 49 127
40 40 118 176
15 25 142 189
23 49 126 136
71 104 110 199
57 184 194 199
42 185 198 199
37 96 184 189
16 28 88 150
139 176 184 187
12 119 140 197
182 192 198 199
79 96 156 172
83 163 166 175
21 95 157 165
87 102 182 192
19 107 180 195
107 162 163 167
4 60 70 126
37 95 154 183
20 165 192 199
112 180 188 199
19 133 161 184
49 142 156 191
53 192 197 198
4 5 80 119
132 146 157 187
15 41 78 120
48 81 91 148
32 68 107 141
27 124 181 187
56 106 178 196
2 4 101 172
61 64 144 177
1 11 34 86
3 3 128 170
56 57 91 121
105 153 167 191
120 148 174 177
19 88 148 164
130 155 168 175
7 84 101 144
7 57 132 189
0 13 137 160
103 112 195 197
56 166 168 199
86 111 116 132
137 186 187 188
36 37 92 166
8 179 195 195
32 65 141 186
32 136 191 198
36 197 198 199
64 86 126 178
8 26 178 199
19 151 184 192
12 32 122 125
4 98 121 139
9 88 102 127
108 179 186 195
33 46 100 141
9 11 68 84
29 110 124 162
106 186 188 195
2 17 195 197
12 118 178 191
22 171 195 196
23 78 97 143
109 173 179 183
37 102 165 193
19 126 184 184
163 189 191 194
29 35 42 157
157 190 199 199
39 80 117 199
18 71 166 167
13 118 149 165
36 119 138 160
134 139 158 168
31 32 160 164
56 143 182 199
46 53 88 154
82 170 195 199
59 136 190 199
24 38 72 125
112 174 182 193
63 156 168 191
3 7 14 116
3 194 195 199
111 157 182 195
109 192 196 198
135 139 161 193
52 54 106 136
42 96 116 194
155 194 199 199
69 73 156 164
106 159 182 185
64 76 176 176
68 132 169 199
70 96 148 157
65 175 181 190
0 102 147 174
32 94 195 197
0 5 141 162
10 15 74 175
164 173 190 192
112 148 197 199
2 23 175 199
33 151 175 195
34 186 193 195
124 130 173 176
61 118 144 186
107 155 196 197
55 170 171 185
10 192 194 197
2 13 147 199
86 190 199 199
6 40 117 157
123 157 161 174
4 38 84 91
120 165 169 176
15 16 189 199
8 51 104 155
14 98 167 199
1 46 152 184
8 13 165 199
24 34 139 176
28 28 189 192
66 191 197 198
82 100 165 195
55 88 97 150
14 126 193 198
153 172 178 184
134 146 148 166
27 96 199 199
1 18 29 134
12 61 195 198
150 153 176 195
129 135 177 197
43 54 106 198
27 99 109 152
26 39 170 192
29 62 157 194
39 92 173 193
91 184 197 197
178 195 197 197
13 120 194 198
39 115 118 180
83 161 180 191
7 31 92 151
19 183 195 199
14 77 125 153
32 108 144 164
27 124 128 188
17 97 149 153
69 121 194 194
99 100 113 197
133 163 197 197
46 106 188 190
67 111 156 156
169 174 190 196
151 154 155 193
36 103 161 182
11 53 179 179
5 35 192 199
103 152 197 197
122 152 178 178
24 25 181 189
12 70 115 193
35 41 42 160
40 40 129 168
30 137 172 193
97 160 179 187
0 50 106 191
6 8 195 195
