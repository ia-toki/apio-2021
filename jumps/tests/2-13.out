fc5005cd102fffc504bc1615010ebf88823d65f3
OK
-1
-1
-1
-1
5
-1
-1
3
3
-1
3
-1
3
-1
-1
1
2
-1
2
1
-1
-1
-1
1
1
4
3
2
-1
1
2
2
-1
2
-1
1
1
3
1
-1
-1
-1
1
-1
2
-1
3
2
2
1
-1
2
-1
1
2
-1
-1
5
1
2
6
-1
-1
-1
-1
-1
1
2
3
-1
-1
-1
1
3
-1
1
-1
1
2
-1
-1
-1
-1
1
3
-1
-1
1
1
-1
-1
-1
2
-1
2
-1
2
1
-1
1
2
-1
1
3
-1
-1
2
-1
-1
-1
-1
-1
2
1
-1
1
3
-1
1
-1
-1
-1
-1
-1
-1
-1
1
-1
4
3
-1
-1
-1
-1
-1
-1
1
-1
1
1
3
-1
2
1
-1
-1
-1
2
-1
3
4
3
-1
1
3
1
-1
-1
2
-1
2
-1
-1
-1
2
1
-1
2
-1
-1
-1
-1
1
-1
2
2
1
1
1
-1
-1
2
-1
-1
-1
-1
1
2
-1
-1
-1
-1
-1
1
1
3
-1
-1
2
-1
